### PR TITLE
update join to avoid fan out and remove disabled

### PR DESCRIPTION
**Pull request by eogilvy12**
### Adwords updates

* resolving [this issue](https://github.com/fishtown-analytics/adwords/issues/13) - the old default was to disable url performance and if you didn't specify to enable it, the package would not run successfully
* because we now want to potentially build the `ad_performance_adapter` from the url report, we don't want the default to be disabled
* add documentation for links to stitch, fivetran (to be added soon) and google's api documentation
* add link to dbt docs
* add instructions for adding the package
* add var to project to specify if you want to use the url performance report or the criteria performance report for the `ad_performance_adapter`. instructions added to read.me 
* this passes on redshift and snowflake

### CODE REVIEW

File: dbt_project.yml [hunk 1]
```diff
@@ -11,11 +11,6 @@ data-paths: ["data"]
 
 models:
   adwords:
-    stitch:
-      base:
-        adwords_url_performance:
-          enabled: false # currently not implemented for all adapters; override this flag if you want to use it.
-    vars:
```
Comment (jthandy) [inline dbt_project.yml:18]: I don't think you meant to delete this line, right?

**Comment by eogilvy12** [comment]
@jthandy this is ready for a second set of eyes! Chris and I are going to be working on adding fivetran support over the next few days but wanted this to fix the fan out first

**Review by jthandy — APPROVED** [review]

File: dbt_project.yml (hunk 1 above)
Comment (eogilvy12) [reply dbt_project.yml:18]: i did but i am interested if you prefer to keep it in - the issue was that this was enabled false as a default in a recent update and yet the version required it for the `adwords_ad_performance_adapter`. this caused all runs to fail unless you enabled it true. now, there are two options for ad performance adapter

Comment (jthandy) [reply dbt_project.yml:18]: 👍

**Pull request merged**